website/src/components/Header/Header.tsx

```diff
@@ -13,6 +13,7 @@ import { UserMenu } from "./UserMenu";
 import { UserScore } from "./UserScore";
 
 function AccountButton() {
+  const { t } = useTranslation();
```

Comment:
ok thank you, one final comment, could you please explicitly use the common namespace?

```js
const { t } = useTranslation("common");
```

This will help with typechecking